src/rpc/client.cpp

```diff
@@ -123,6 +123,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "gettxout", 1, "n" },
     { "gettxout", 2, "include_mempool" },
     { "gettxoutproof", 0, "txids" },
+    { "gettxoutsetinfo", 0, "hash_type" },
```

Comment:
87dd07e0382c6398d118d88c88b0c9ef24251e37

why is this needed? Looks like a str to me

Reply:
Indeed, this should be dropped (try: `bitcoin-cli gettxoutsetinfo none`, it works without this line).

Reply:
done